### PR TITLE
minor update

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md [hunk 1]
```diff
@@ -67,7 +67,11 @@ server machine of your choice:
 1. Install the latest version of [curl](https://curl.haxx.se/download.html).
 1. Install [Python](https://www.python.org/) version 3.7 or later.
 1. Install the latest version of [jq](https://github.com/stedolan/jq/wiki/Installation) command-line JSON parser. This is required for running the fields.sh batch file.
```
Comment (JainM6) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md:69]:
```suggestion
1. Install the latest version of [jq](https://github.com/stedolan/jq/wiki/Installation) command-line JSON parser. This is required for running the `fields.sh` batch file.
```

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md [hunk 2]
```diff
@@ -20,11 +20,11 @@ To set up the AWS Observability solution using Terraform, complete the followin
 Additional parameter overrides are available in an appendix section for [Source](#override-source-parameters) and [App Content](#override-app-content-parameters).
 
 :::note
-If you have already set up the solution with CloudFormation in the past and want to move to Terraform, we recommend that you:
+If you have already set up the solution with CloudFormation in the past and want to move to Terraform, we recommend you to follow the below instruction:
```
Comment (JainM6) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md:23]:
```suggestion
If you have already set up the solution with CloudFormation in the past and want to move to Terraform, we recommend you follow the below instructions:
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Formatting.

File: docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md [hunk 3]
```diff
@@ -66,8 +66,12 @@ server machine of your choice:
     ```
 1. Install the latest version of [curl](https://curl.haxx.se/download.html).
 1. Install [Python](https://www.python.org/) version 3.7 or later.
-1. Install the latest version of [jq](https://github.com/stedolan/jq/wiki/Installation) command-line JSON parser. This is required for running the fields.sh batch file.
-
+1. Install the latest version of [jq](https://github.com/stedolan/jq/wiki/Installation) command-line JSON parser. This is required for running the `fields.sh` batch file.
+1. Install Sumo Logic Python SDK using the following command. Click [here](https://pypi.org/project/sumologic-sdk/) to know more.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/deploy-with-terraform.md:70]:
```suggestion
1. Install Sumo Logic Python SDK using the following command. Click [here](https://pypi.org/project/sumologic-sdk/) to learn more.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
grammar

**Review by JainM6 — APPROVED** [review]
LTGM

**Review by kimsauce — APPROVED** [review]

**Pull request merged**